public/index.html

```diff
@@ -32,6 +32,28 @@
   </head>
   <body>
     <noscript>You need to enable JavaScript to run this app.</noscript>
+
+    <div id="preloader">
```

Comment:
Hey @magicmatatjahu 

Just wondering, how come you decided to add the reloader here inside the html vs inside the react component?

Was it a performance thing? You wanted to show the loader ASAP, whilst the app boots up?

My initial thought would be to do it in the app.jsx or somewhere, but yeah just wanted to know your thought process :)

Reply:
> You wanted to show the loader ASAP, whilst the app boots up?

Yes :) If we will add loader to the main component, then it will be rendered when script will be loaded and that it will show immidiatelly :)

Reply:
Yeah nice touch never thought of that before, loader looks great btw. Cool!